modular_skyrat/modules/deforest_medical_items/code/medstation_designs/blood.dm

```diff
@@ -0,0 +1,109 @@
+/datum/design/organic_bloodbag_aplus
+	name = "A+ Blood Pack"
+	id = "organic_bloodbag_aplus"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 100)
+	build_path = /obj/item/reagent_containers/blood/a_plus
+	category = list(
+		RND_CATEGORY_INITIAL,
+		RND_CATEGORY_DEFOREST_BLOOD,
+	)
+
+/datum/design/organic_bloodbag_aminus
+	name = "A- Blood Pack"
+	id = "organic_bloodbag_aminus"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 100)
+	build_path = /obj/item/reagent_containers/blood/a_minus
+	category = list(
+		RND_CATEGORY_INITIAL,
+		RND_CATEGORY_DEFOREST_BLOOD,
+	)
+
+/datum/design/organic_bloodbag_bplus
+	name = "B+ Blood Pack"
+	id = "organic_bloodbag_bplus"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 100)
+	build_path = /obj/item/reagent_containers/blood/b_plus
+	category = list(
+		RND_CATEGORY_INITIAL,
+		RND_CATEGORY_DEFOREST_BLOOD,
+	)
+
+/datum/design/organic_bloodbag_bminus
+	name = "B- Blood Pack"
+	id = "organic_bloodbag_bminus"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 100)
+	build_path = /obj/item/reagent_containers/blood/b_minus
+	category = list(
+		RND_CATEGORY_INITIAL,
+		RND_CATEGORY_DEFOREST_BLOOD,
+	)
```

Comment:
I really don't understand why provide Rh+ varieties at the same cost as Rh- ones. At that point why have them?